script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -427,17 +470,21 @@ It could be that the one from before was the last of the dead collecting food. F
 [sha:552839905214f32b5de491d43def80739bf0fa5e]{
 -- Page 462, Offset 17722.
 -- そんな理由で機嫌を損ねられると、なんか、困る。
-Hearing that's the reason for her being upset is... a bit embarassing.
+// We sure about embarrassing as the reading of 困る here?
+// Not like he's bothered by it and wants to put it right?
```

Comment:
This is how it reads to me given his next line. He's bothered she's disappointed and wants to keep going.